src/patternfly/upgrade-examples/BadgeUpgradeExamples/styles.scss

```diff
@@ -0,0 +1,28 @@
+@import "../../patternfly-utilities";
```

Comment:
Are these styles necessary?

Reply:
I dont think so, it is created automatically, going to remove it